test/schema/750-test.test.fetch.sql

```diff
@@ -0,0 +1,17 @@
+ALTER PROCEDURE test.[test.fetch]
+    /*{
+        "data.content": 1
+    }*/
+    @ngramTest [test].[ngramTT] READONLY
+AS
+
+SELECT 'data' AS resultSetName
+
+SELECT
+    t.id, t.txt
+FROM
+    test.test t
+JOIN
+    test.testSearch(@ngramTest, 1) s
+ON
+    s.id = t.id
```

Comment:
The proper way to search is something like this:
```sql
WHERE t.id IN (SELECT id FROM test.testSearch(@ngramTest, 1))
```
instead of `JOIN`.
I think using `JOIN` will result in:
1) Multiple rows with the same `id` and `text` in the result
2) Returning rows when only one ngram is matched. I.e. if your data contains 'abcde' and you search for 'xyabc', you will still get matches.

Can you add this kind of test before fixing the procedure to see if that's the case? 
Then fix the procedure and check that this test fails to find records.

Reply:
[done](https://github.com/softwaregroup-bg/ut-port-sql/commit/33aa5f95966ab566e8cae9f659b904429c8c2831)

Reply:
it works both ways because the `search` function groups records by id: https://github.com/softwaregroup-bg/ut-port-sql/blob/betterDocParseError/sql.js#L311-L312

Reply:
I can't see the additional test. Please add search for things like 'abcde', 'abcdx', 'xyabc', 'abc cde'.
Searching only strings with length 3 does not catch all the cases.

Reply:
> I can't see the additional test. Please add search for things like 'abcde', 'abcdx', 'xyabc', 'abc cde'. Searching only strings with length 3 does not catch all the cases.

[done](https://github.com/softwaregroup-bg/ut-port-sql/commit/41cdda9a2db3dbee2db386b15f7225015c9df0db)